spec/components/parameters/organizationId.yaml

```
@@ -0,0 +1,6 @@
+name: Organization-Id
+in: header
+description: Organization identifier (if not specified, the default organization will be used)
```

```suggestion
description: Organization identifier in scope of which need to perform request (if not specified, the default organization will be used)
```

Not sure about the sentence itself, but the idea is to describe the parameter more verbose and not to have description is just the same as parameter name